simple-calc/modules/Resources/CSharp/Figma/Import-UI-from-Figma.md

```diff
@@ -32,7 +39,7 @@ uid: Workshop.SimpleCalc.Figma.CSharp
 [!code-csharp[MaterialFontsOverride.cs](MaterialFontsOverride.cs)]
 </details>
 
-7. Now we need to prepare our UI with the Binding expressions that we will need in the App Architecture module. First let's add the `DataContext` to the page. To do so add `.DataContext(new TempDataContext(), (page, vm) => page` before the `.Content` definition. Ensure to properly terminate the DataContext with a closing `)` preceding the semicolon at the end of the page's code.
+8. Now we need to prepare our UI with the Binding expressions that we will need in the App Architecture module. First let's add the `DataContext` to the page. To do so add `.DataContext(new TempDataContext(), (page, vm) => page` before the `.Content` definition. Ensure to properly terminate the DataContext with a closing `)` preceding the semicolon at the end of the page's code.
 
 ```csharp
```

Comment:
This entire code-block should be indented, as it's nested under a number bullet point.